assets/js/src/core/modules/data-object/listing/batch-actions/batch-edit-modal/batch-edit-modal.tsx

```diff
@@ -150,12 +156,36 @@ export const BatchEditModal = ({ batchEditModalOpen, setBatchEditModalOpen }: Ba
 
           return response.data?.jobRunId
         },
+        refreshGrid,
         // @todo change that to a more generic context
         assetContextId: id
       }))
     }
   }
 
+  const availableDropdownList = getAvailableColumnsDropdown(onColumnClick).menu.items
+
+  const getFilteredTypes = (column: any): object[] => {
+    return column?.children?.filter((child: any) => {
```

Comment:
I think we still miss here that we only show the fields that are really editable?
In the column should be a prop like editable.